src/pages/training.js

```diff
@@ -1,52 +1,26 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
 import { graphql } from 'gatsby'
 
 import { Grid } from '../components/grid'
 import Layout from '../components/layout'
 import Statement from '../components/Common/Statement'
 import Approach from '../components/Training/Approach'
 import Courses from '../components/Training/Courses'
-import Modal from '../components/Training/Modal'
 import CaseStudyPreview from '../components/Common/CaseStudyCards/CaseStudyPreview'
 import GetInTouch from '../components/Common/GetInTouch'
 import Head from '../components/Common/Head'
 import GreyBackground from '../components/Common/GreyBackground'
 
 const TrainingPage = ({ data: { contentfulTrainingPage: content } }) => {
-  const [modalContent, setModalContent] = useState(null)
   const trainingApproachesContent = [
     content.trainingApproachContent1.trainingApproachContent1,
     content.trainingApproachContent2.trainingApproachContent2,
     content.trainingApproachContent3.trainingApproachContent3
   ]
 
-  useEffect(
```

Comment:
any reason for not having ported this to `trainingCourseModal`? We lost the functionality to hitting 'escape' to close the modal, didn't we?

Reply:
We can discuss this in the stand-up as we need to discuss the implementation following our changes

Reply:
@antoniocapelo We figured that `useEffect` wouldn't work for this new `trainingCourseModal` implementation.

Now that each modal is actually it's own page, the transition needs to happen on page load, but [`useEffect` only fires after the page is painted](https://reactjs.org/docs/hooks-reference.html#timing-of-effects)
> Unlike `componentDidMount` and `componentDidUpdate`, the function passed to useEffect fires **after** layout and paint, during a deferred event.

If I'm missing something, and we can still use this, do let me know :+1: 

Assuming that `useEffect` wasn't what we needed, we started trying a different method than React hooks - i.e. [this gatsby plugin](https://www.gatsbyjs.org/packages/gatsby-plugin-page-transitions/). But it seemed prudent to wait until the re-implementation of the modal, which we have already scoped to [a separate ticket](https://trello.com/c/0j78BthW/458-revisit-modal-implementation).

I discussed this with Beatriz and James, who said they were happy for this PR to go in without the animation, as it wasn't part of the original designs anyway. I still think we should put it back in, but just that it might be worth leaving to another ticket? What do you reckon?